osquery/tables/yara/yara.cpp

```diff
@@ -21,6 +27,23 @@
 #include <yara.h>
 
 namespace osquery {
+
+// After a large scan of many files, the memory allocation could be
+// substantial.  free() may not return it to operating system, but
+// rather keep it around in anticipation that app will reallocate.
+// Call malloc_trim() on linux to try to convince it to release.
+#ifdef LINUX
+FLAG(bool,
+     yara_malloc_trim,
+     true,
+     "Call malloc_trim() after YARA scans (linux)");
+#endif
+FLAG(uint32,
+     yara_intrafile_sleep,
+     50,
+     "Time in ms to sleep after scan of each file (default 50). Helps reduce "
```

Comment:
nitpick, there's punctuation in this flag's description but not `yara_malloc_trim`. Looking at "all of the flags" it seems most common to not have punctuation, for whatever reason this precedent was set, hehe.